### PR TITLE
Fix and add additional console formatter environment checks.

### PR DESCRIPTION
**Pull request by ndm2**
Guess they wouldn't be pretty, but should there maybe be some tests for the environment check?

### CODE REVIEW

File: src/Error/Debug/ConsoleFormatter.php [hunk 1]
```diff
@@ -66,7 +66,10 @@ public static function environmentMatches(): bool
         // Windows environment checks
         if (
             DIRECTORY_SEPARATOR === '\\' &&
-            (!(bool)env('ANSICON') || env('ConEmuANSI') !== 'ON')
+            strpos(strtolower(php_uname('v')), 'windows 10') === false &&
```
Comment (markstory) [inline src/Error/Debug/ConsoleFormatter.php]: Do we need this? If we are running on windows and in bash.exe can't we safely assume we are in a bash emulator?

**Review by markstory — COMMENTED** [review]
I thought about tests but getting them to pass consistently across windows and linux felt like they would be duplicating the implementation in skip logic

File: src/Error/Debug/ConsoleFormatter.php (hunk 1 above)
Comment (ndm2) [reply src/Error/Debug/ConsoleFormatter.php]: That's actually for two different scenarios, as of Windows 10 the console host supports colors out of the box, the bash check would be for earlier Windows versions.

**Comment by ndm2** [comment]
> I thought about tests but getting them to pass consistently across windows and linux felt like they would be duplicating the implementation in skip logic

Yeah, that's what I thought too.

**Pull request merged**